### PR TITLE
build script for package go-hclog

### PR DESCRIPTION
**Pull request by vikasgupta8**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: g/go-hclog/go-hclog_ff2cf002a8dd_ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: go-hclog
+# Version	: ff2cf002a8dd
```
Comment (seth-priya) [inline g/go-hclog/go-hclog_ff2cf002a8dd_ubi8.4.sh:4]: is this for the top-of-tree or for an older version? If its top-of-tree, can we remove the commit # from the script name?

Comment (vikasgupta8) [reply g/go-hclog/go-hclog_ff2cf002a8dd_ubi8.4.sh:4]: The version was mentioned in the list is as "**v0.0.0-20180709165350-ff2cf002a8dd**". It's commit id is ff2cf002a8dd. 
Could you please advise if I need to use v.0.0.0 in the script name?

Comment (seth-priya) [reply g/go-hclog/go-hclog_ff2cf002a8dd_ubi8.4.sh:4]: can we test it for the default branch and if it works, remove the version altogether from the script name?

Comment (vikasgupta8) [reply g/go-hclog/go-hclog_ff2cf002a8dd_ubi8.4.sh]: Removed second script. First script is working with main(master) branch also.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**